CMakeLists.txt

```diff
@@ -0,0 +1,242 @@
+cmake_minimum_required(VERSION 3.10)
+project(WatchySim VERSION 1.0.0 LANGUAGES CXX)
+
+# Set C++ standard
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+set(CMAKE_CXX_EXTENSIONS OFF)
+
+# Options for different watch faces
+option(BUILD_7SEG "Build 7-Segment watch face" ON)
+option(BUILD_ANALOG_GABEL "Build Analog Gabel watch face" OFF)
+option(BUILD_DOS "Build DOS watch face" OFF)
+option(BUILD_MACPAINT "Build MacPaint watch face" OFF)
+option(BUILD_MARIO "Build Mario watch face" OFF)
+option(BUILD_POKEMON "Build Pokemon watch face" OFF)
+option(BUILD_POWERSHELL "Build PowerShell watch face" OFF)
+option(BUILD_TETRIS "Build Tetris watch face" OFF)
+option(BUILD_BASIC "Build Basic watch face" OFF)
+
+# Find SDL2
+find_package(PkgConfig REQUIRED)
+pkg_check_modules(SDL2 REQUIRED sdl2)
+
+# Include directories
+include_directories(
+    ${SDL2_INCLUDE_DIRS}
+    ${CMAKE_SOURCE_DIR}/WatchySim
+    ${CMAKE_SOURCE_DIR}/WatchySim/arduino
+)
+
+# Define macros
+add_definitions(-DLINUX_BUILD)
+add_definitions(-DSDL2_BACKEND)
+add_definitions(-DWATCHY_SIM)
+
+# Core source files
+set(CORE_SOURCES
+    WatchySim/WatchySim_SDL2.cpp
```

Comment:
Can you have these keep the same names, but put them in /ports/linux (and the Windows ones in /ports/windows)?

Reply:
Actually, after further review it's possible that these files don't even need to exist separately.